lib/node_modules/@stdlib/math/base/special/max/benchmark/c/native/benchmark.c

```diff
@@ -1,7 +1,7 @@
 /**
```

Comment:
Keep both files `c/benchmark.c` and `c/native/benchmark.c`

Reply:
Resolved